integration/docker/Dockerfile

```diff
@@ -73,6 +85,8 @@ RUN if [ ${ALLUXIO_USERNAME} != "root" ] \
       mkdir -p /journal && \
```

Comment:
The command to adduser is also different between debian and alpine.

Reply:
@apc999 Maybe this is the problem you're running into?